README.rst

```diff
@@ -94,7 +94,10 @@ Preparing the Host Cloud Environment
 
 #. Upload an ipxe-boot image for the baremetal instances::
 
-    glance image-create --name ipxe-boot --disk-format qcow2 --container-format bare < ipxe/ipxe-boot.qcow2
+    glance image-create --name ipxe-boot --disk-format qcow2 --property os_shutdown_timeout=5 --container-format bare < ipxe/ipxe-boot.qcow2
```

Comment:
Cool, this is a nice tweak to avoid having to make the config change on the host cloud.